esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -3,11 +3,16 @@
     UTC mode = True
 
 [task parameters]
-    metric = radiation_budget
+    small = modes_of_variability, multimodel_products
+    medium = ensclus, heatwaves_coldwaves
 
 [scheduling]
     [[graph]]
-        R1 = install_cold => configure => process<metric>
+        R1 = """
+             install_cold => configure
+             configure:succeed => process<small>
+             configure:succeed => process<medium>
```

Comment:
It is not necessary to include `:succeed`, since this is the default.

The `fast` and `medium` recipes should run in difference cycles / at different times.

Reply:
Yes, I know - but at the moment they are not reunning successfully so I am trying to solve that problem first.
I will make this a draft so it is more obvious.

Reply:
addressed in c9c2e2fd22c25a920b4de7c1e5bf784e04641ed0

Reply:
I think this was addressed in 9cab64c?